client/lib/utils.js

```diff
@@ -142,6 +142,14 @@ Utils = {
                 return 'labels.' + labels[labelId] + '.' + key;
             }
         }
+    },
+
+    // Extract userId from user object
+    getUserId: function(user) {
```

Comment:
I have removed this method. Now developers can obtain user ID using `this.user._id`. Also in last commit I have replaced `this.user._id` with `this.userId` to make it more similar to original code, If you are OK with `this.user._id`, you can ignore last commit. (Sorry for my weak English grammar ;))